cmd/check-reaper/Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM golang:1.13-alpine AS builder
+FROM golang:1.14.0 AS builder
```

Comment:
```suggestion
FROM golang AS builder
```

Reply:
let's build with the latest golang... this ensures the newest fixes and all

Reply:
isn't it bad practice to build with `latest`?